prisma-fmt/tests/text_document_completion/scenarios/extended_indexes_operators_postgres_gist/result.json

```diff
@@ -0,0 +1,13 @@
+{
+  "isIncomplete": false,
+  "items": [
+    {
+      "label": "InetOps",
+      "kind": 13
+    },
+    {
+      "label": "raw",
```

Comment:
Nitpick: should we return `raw("")` (as a snippet, with the cursor inside the quotes) here instead?

Reply:
I don't know actually. I'm using the type `Function` here, and I hope the editor automatically adds `()` when completing.

Reply:
Would be nice indeed, added a comment about how to that here https://github.com/prisma/prisma-engines/pull/2993#discussion_r902566172

Reply:
The editor will not add anything here from experience, only change the icon for the completion

Reply:
This should work:

```rust
completions.items.push(CompletionItem {
    label: "raw".to_string(),
    insert_text: Some(r#"raw("$0")"#.to_string()),
    insert_text_format: Some(InsertTextFormat::SNIPPET),
    kind: Some(CompletionItemKind::FUNCTION),
    ..Default::default()
});
```

Reply:
Looks good